src/modules/users/index.ts

```diff
@@ -1,5 +1 @@
 export * from './users.module';
```

Comment:
being a common module, it will be used by many others, so let's keep `'./services/users.interface` exported